build/Targets/Imports.targets

```diff
@@ -35,6 +35,11 @@
     <_NETCoreSdkIsPreview>false</_NETCoreSdkIsPreview>
   </PropertyGroup>
 
+  <!-- Settings for Visual Studio packages -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.SDK.Analyzers" Version="$(MicrosoftVisualStudioSDKAnalyzersVersion)" PrivateAssets="all" Condition="'$(RoslynProjectType)' == 'Vsix'" />
```

Comment:
I think we should apply this VSIX directly in the VSIX project files.

Reply:
This isn't a VSIX. It's an analyzer nuget package.
Ideally (from my perspective) it would be applied to any project that contains VS extension code. If all such code is directly in your VSIX projects, then I support your suggestion.

Reply:
Corrected my typo: s/VSIX/package 

Our extension code is more spread out than just the VSIX projects.

Reply:
> Our extension code is more spread out than just the VSIX projects.

Then I think the analyzers should spread to all the relevant projects. Do you agree?

Reply:
> Then I think the analyzers should spread to all the relevant projects. Do you agree?

I agree, but I don't think it would be a practical change for this pull request.

Reply:
Didn't @jasonmalinowski make the comment that there isn't any code in the VSIX projects themselves? If so, enabling the analyzers on those code-less projects won't buy you anything.

Reply:
Our shipping code doesn't. We have a test VSIX or two, which is then why yes, I was surprised by that one change of .Result and not the rest of them.

Reply:
> I think we should apply this package directly in the VSIX project files.

Fixed in 20cb7aee69fe75c72b6931a84932d345a6a2175d